src/main/java/org/ays/auth/model/request/AysUserUpdateRequest.java

```diff
@@ -17,7 +17,8 @@
 /**
  * Request object for updating a user's details.
  * <p>
- * This class encapsulates the data required to update an existing user's details.
+ * This class encapsulates the data required to update an existing user's
+ * details.
```

Comment:
Özel bir formatter kullanıyosun sanırım, herkes default IntellIJ formatter'ını kullanıyor bu sebeple burayı eski hâline alabiliriz
```suggestion
 * This class encapsulates the data required to update an existing user's details.
```